packages/nimble-components/src/table/models/table-update-tracker.ts

```diff
@@ -132,7 +132,8 @@ export class TableUpdateTracker<
                 changedColumnProperty,
                 'currentFractionalWidth',
                 'currentPixelWidth',
-                'minPixelWidth'
+                'minPixelWidth',
+                'resizingDisabled'
```

Comment:
While I realize this is fairly innocuous, I'm curious if there's a practical reason why we have to track this change? I would have expected we only need to track property changes that can result in a visual width change of the columns.